fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -187,35 +186,26 @@ void init() throws IOException {
     }
 
     @Override
-    public void writeTo(final RdfStream rdfStream, final Class<?> type,
+    public void writeTo(final RdfNamespacedStream nsStream, final Class<?> type,
                         final Type genericType, final Annotation[] annotations,
                         final MediaType mediaType,
                         final MultivaluedMap<String, Object> httpHeaders,
                         final OutputStream entityStream) throws IOException {
 
-        try {
-            final RdfStream nsRdfStream = new NamespaceRdfContext(rdfStream.session());
-
-            rdfStream.namespaces(nsRdfStream.namespaces());
-
-            final Node subject = VIEW_HELPERS.getContentNode(rdfStream.topic());
-
-            final Model model = rdfStream.asModel();
+        final Node subject = VIEW_HELPERS.getContentNode(nsStream.stream.topic());
```

Comment:
It seems like an anit-pattern for clients of `RdfNamespacedStream` to need to be aware of the internals of `RdfNamespacedStream.stream` in order to get `RdfNamespacedStream.stream.topic`. 
If however, `RdfNamespacedStream` implemented `RdfStream`, clients could instead simply call `RdfNamespacedStream.topic()`.